seed/ts-express/seed.yml

```diff
@@ -50,6 +50,9 @@ fixtures:
         includeUtilsOnUnionMembers: true
         includeOtherInUnionTypes: true
       outputFolder: union-utils
+    - customConfig:
+        useJsFileExtensionsOnImports: true
+      outputFolder: js-file-extensions
```

Comment:
should I add this test under `exhaustive` or somewhere else?

Reply:
`exhaustive` works